third_party/npm_@trezor_connect-web/LICENSE

```diff
@@ -0,0 +1,50 @@
+# TREZOR REFERENCE SOURCE LICENSE (T-RSL)
+
+This license governs use of the accompanying software. If you use the software,
+you accept this license. If you do not accept the license, do not use the
+software.
+
+## 1. Definitions
+
+The terms "reproduce," "reproduction" and "distribution" have the same meaning
+here as under U.S. copyright law.
+
+"You" means the licensee of the software.
+
+"Your company" means the company you worked for when you downloaded the
+software.
+
+"Reference use" means use of the software within your company as a reference,
+in read only form, for the sole purposes of debugging your products,
+maintaining your products, or enhancing the interoperability of your products
+with the software, and specifically excludes the right to distribute the
+software outside of your company.
```

Comment:
Seems like using this might mean we're in violation of their license since we'd be distributing it outside the company. Am I reading this correctly?

Reply:
just noting this is already the case for the current version AFAICT - https://github.com/trezor/connect/blob/develop/LICENSE.md. however we should still check in on it.

Reply:
update: @bbondy got written permission from the CTO of Satoshi Labs to use this library in Brave when it was first added, so I think we are good to go.